.gitignore

```diff
@@ -8,3 +8,5 @@
 ext/openssl/mkmf.log
 ext/openssl/Makefile
 ext/openssl/extconf.h
+.DS_Store
+.idea/
```

Comment:
Unrelated changes checked in?

Reply:
It's never a change by itself but removing Intelli-J and macOS files from being tracked is something I do with first contribution.  Some projects find it helpful, some do not

Reply:
IMO these are a better fit in `~/.config/git/ignore` rather than in .gitignore of every project.